pytest-selenium/tests/test_attribution_pages.py

```diff
@@ -0,0 +1,102 @@
+import unittest
+from time import sleep
+
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+
+class TestAttributionpages(unittest.TestCase):
+    def setUp(self):
+        self.browser = webdriver.Chrome()
+        self.browser.implicitly_wait(3)
+    def teardown(self):
+        self.browser.quit()
+
+
+    def offscreen_click(self, element):
+        """Clicks an offscreen element.
+        Clicks the given element, even if it is offscreen, by sending the ENTER key.
+        Returns the element.
+        """
+        # We actually navigate using the ENTER key because scrolling the page can be flaky
+        # https://stackoverflow.com/a/39918249
+        element.send_keys(Keys.ENTER)
+        return element
+
+
+#c476304 citation/attribution section collapses by default when you navigate to a new page
+    
+    def test_c476304_attribution_pages_navigation(self):
+        #page_url = base_url + "/books/college-physics/pages/preface"
+        page_url = "https://rex-web.herokuapp.com/books/college-physics/pages/preface"
+        attribute_xpath_locator = "//details[contains(@class,'Attribution__Details')]"
+        self.browser.get(page_url)
+        attribute_element = self.browser.find_element_by_xpath(attribute_xpath_locator)
+
+        #Expand attribution
+        attribute_element.click()
+        attribute = attribute_element.get_attribute("open")
+        print(attribute)  #true
+
+
+        #Navigate to Next page and verify attribution is closed by default
+        Next_element_locator = self.browser.find_element_by_css_selector("a[aria-label='Next Page']")
+        self.offscreen_click(Next_element_locator)
+        sleep(1)
```

Comment:
We need to figure out a way to not use explicit sleeps. We want to use implicit sleeps. The same way a page is detected as loaded we can add for a page region. We can also look and see if the region is displayed.

Reply:
Updated with region lookup in 2 test cases. C476304 still works only with sleep. I will look into this when reworking the script for TOC layout changes